Formula/lib/libnxml.rb

```diff
@@ -16,19 +16,16 @@ class Libnxml < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "63bb969c6efc96d3bc6c97c4e8faa0dc918ed41bdf9d55e25f372f33b4c4d78e"
   end
 
-  # Regenerate `configure` to avoid `-flat_namespace` bug.
-  # None of our usual patches apply.
```

Comment:
If this is no longer true then we should probably stop calling `autoreconf`

Reply:
the github source tarball does not have configure file, so we still need `autoreconf`

same situation with #163589